.github/CODEOWNERS

```diff
@@ -5,20 +5,44 @@
 
 *       @department-of-veterans-affairs/frontend-review-group
 
-# VSA
-src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-of-veterans-affairs/vsa-frontend
-src/applications/find-forms @department-of-veterans-affairs/vsa-frontend
-src/platform/site-wide @department-of-veterans-affairs/vsa-frontend
-src/applications/yellow-ribbon @department-of-veterans-affairs/vsa-frontend
-src/applications/personalization @department-of-veterans-affairs/vsa-frontend
-src/applications/letters @department-of-veterans-affairs/vsa-frontend
-src/applications/disability-benefits @department-of-veterans-affairs/vsa-frontend
-src/platform/user/profile/vet360 @department-of-veterans-affairs/vsa-frontend
+# VSA ----
+
+# Authenticated Experience
+src/applications/personalization @department-of-veterans-affairs/vsa-authd-exp-frontend @department-of-veterans-affairs/vsa-ebenefits-frontend
+src/platform/user/profile/vet360 @department-of-veterans-affairs/vsa-authd-exp-frontend @department-of-veterans-affairs/frontend-review-group
```

Comment:
And in this case the entire frontend-review-group (which is VSP?) will be notified of any changes to `src/platform/user/profile/vet360`? that makes sense since it lives in platform. As long as they are cool with that :)

Reply:
Yep! I did the same for other `platform/` directories that PW works in